portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/BlogsEvents.testcase

```diff
@@ -521,7 +507,7 @@ definition {
 		task ("Start Har recording") {
 			ProxyUtil.startHarRecording("BlogViewed");
 
-			Pause(locator1 = "5000");
+			ACUtils.waitForSendingRequest();
```

Comment:
Hey @calvinctvo, I think you could put this macro for after the iteration in DXP. I think you could put. What do you think?

```
task ("Wait for sending requests") {
	ACUtils.waitForSendingRequest();
}
```

After

```
task ("Go to the site page to assert the blog entry is displayed but don't visit it") {
	ACUtils.navigateToSitePage(
		pageName = "AC Page",
		siteName = "Site Name");
		
	BlogsEntry.viewEntryPG(
		entryContent = "This is the blog test for AC",
		entryTitle = "Blogs AC Title");
}
```